front_end/panels/ai_chat/ui/__tests__/ChatViewAgentSessions.test.ts

```
@@ -344,6 +344,26 @@ describe('ChatView visibility rules: agent-managed tool calls/results are hidden
     document.body.removeChild(view);
   });
 
+  it('shows configurable agent tool error in chat when execution fails', async () => {
+    const view = document.createElement('devtools-chat-view') as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)